scripts/us_cdc/500_places/download_bulk.py

```diff
@@ -33,8 +33,10 @@
 from absl import flags
 from absl import app
 
-_MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(os.path.join(_MODULE_DIR, '../../../util/'))
+_MODULE_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)),
```

Comment:
Pls keep the variable names close to the value.
Can add a new variable for _OUTPUT
```
_MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
sys.path.append(os.path.join(_MODULE_DIR, '../../../util/'))

_GCS_OUTPUT_DIR = os.path.join(_MODULE_DIR, 'gcs_output')
```

```